aarnet-dev_playbook.yml

```diff
@@ -0,0 +1,91 @@
+- hosts: pawsey_galaxy_server
```

Comment:
aarnet dev equivalent

Reply:
oops

Reply:
this and vars_files needs updating.  Looks good otherwise